### PR TITLE
Post & Timeline Tab Switch components

### PR DESCRIPTION
**Pull request by tomli380576**

### CODE REVIEW

**Comment by tomli380576** [comment]
I will resolve the conflicts today don't worry

**Review by jackie-ou — COMMENTED** [review]
- Are the comments for remove during production supposed to be gone? 
- We have been doing imports for the fonts in their respective CSS files. Not all components use Poppins so might cause unintended issues(?)

**Comment by tomli380576** [comment]
> * Are the comments for remove during production supposed to be gone?
> * We have been doing imports for the fonts in their respective CSS files. Not all components use Poppins so might cause unintended issues(?)

- The remove during production chunks will be removes once this pr is approved. Those are mock data 
- I put Poppins in global import because all the design files we haven uses that font. If we don't want it to be global I can move it to a css module

**Comment by samuelili** [comment]
We can merge this for now, but I'm not too sure if we want to use material-ui and instead write our own components. That really depends on what the designers end up producing.

File: pages/Timeline.tsx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+              </TabPanel>
+            </div>
+          </Slide>
+        </Box>
+      </div>
+    </ThemeProvider>
+  );
+}
+
+export default TimeLineAndExploreTabGroup;
```
Comment (samuelili) [inline pages/Timeline.tsx:211]: It's a huge file but I assume that we'll separate it out later!

File: pages/_app.tsx [hunk 2]
```diff
@@ -9,8 +9,6 @@ import { authOptions } from './api/auth/[...nextauth]';
 export async function getServerSideProps(context: GetServerSidePropsContext) {
   const session = await getServerSession(context.req, context.res, authOptions);
 
-  console.log('hi');
```
Comment (samuelili) [inline pages/_app.tsx:12]: hi!

**Review by samuelili — APPROVED** [review]
epic

**Pull request merged**